sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -411,6 +413,41 @@ class SparkSession private(
    |  Everything else  |
    * ----------------- */
 
+  /**
+   * Executes given script and return the result of the last statement.
```

Comment:
let's also mention that it returns an empty DataFrame if the script contains no query statement.

Reply:
Done.